core/Cargo.toml

```diff
@@ -66,8 +65,6 @@ async-wasm-client = [
 
 [dev-dependencies]
 serde_json = "1.0"
-tokio = { version = "1.16", features = ["macros", "rt"] }
```

Comment:
revert this

Reply:
If we revert this, after adding `unused_crate_dependencies`:

```rs
bear@cypress ~/c/j/core (bear-fix-unused-deps)> cargo test
   Compiling tokio v1.24.2
warning: unused macro definition: `cfg_async_client`
  --> core/src/macros.rs:29:14
   |
29 | macro_rules! cfg_async_client {
   |              ^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_macros)]` on by default

warning: `jsonrpsee-core` (lib) generated 1 warning
   Compiling jsonrpsee-core v0.16.2 (/home/bear/coding/jsonrpsee/core)
error: external crate `tokio` unused in `jsonrpsee_core`: remove the dependency or add `use tokio as _;`
  --> core/src/lib.rs:27:1
   |
27 | //! Shared utilities for `jsonrpsee`.
   | ^
   |
note: the lint level is defined here
  --> core/src/lib.rs:30:9
   |
30 | #![deny(unused_crate_dependencies)]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^

warning: `jsonrpsee-core` (lib test) generated 1 warning (1 duplicate)
error: could not compile `jsonrpsee-core` due to previous error; 1 warning emitted
```

That's why I add the `tokio/macros` to the `http-helpers` feature.

Reply:
tokio/macros is correct but `rt` is just a dev-dependency IIRC